pages/index.tsx

```diff
@@ -1,64 +1,28 @@
 import Head from 'next/head'
 import Image from 'next/image'
 
-import styles from '@/pages/index.module.css'
+import Header from '@/components/header/header';
+import SearchBar from '@/components/searchBar/searchBar';
+import SearchResults from '@/components/searchResults/searchResults';
+
+import styles from '@/pages/index.module.scss'
 
 export default function Home() {
```

Comment:
Architecture Suggestion: consider a directory in pages for Home, e.g pages/Home to reflect what this actually is